### PR TITLE
Portfolio:  contact form section

### PR DESCRIPTION
**Pull request by MPDADDY**
- [ ] Added a contact form in the footer section.
- [ ] Added a submit button.
- [ ] Getting started in the readme has been set properly.
- [ ] fixed misalignment bags in the footer section.
- [ ] Corrected the form action link.
- [ ] Added a max length of 500 to the textarea.
- [ ] Added a name attribute to the inputs and textarea.

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @MPDADDY, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the get started section setup properly, kindly describe to the user how to localize your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Shisui6 — COMMENTED** [review]
Hi @MPDADDY, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛠️

You have submitted a project, with an unprofessional readme file. Please make sure to go through your readme file and fill out all sections with information for your project. Please also make sure to remove the section descriptions (lines that start with >) as they are just comments which are not supposed to be in the final version 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -188,75 +188,115 @@ <h4 class="colorh4">CANOPY</h4>
...
+
+<!---contact me article-->
+
+        <footer>
+            <section class="footer">
+              <h2>Contact me</h2>
+              <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                I'd love to help with it.
+              </p>
+              <form action="https://formspree.io/f/{form_id}" method="post">
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] Kindly makes sure that the form are  submitted successfully with valid input values. When I try to submit the form with values I got the following error related with the `{form_id}`, So to fix the issues Kindly use your own `{form_id}` by creating in the [formspree service](https://formspree.io/html).
Capture

File: index.html [hunk 2]
```diff
@@ -188,75 +188,115 @@ <h4 class="colorh4">CANOPY</h4>
...
+                         <input id="name"  maxlength="30" placeholder="Lorem ipsum" type="text" required>
+                     </label>
+                 </li>
+                 <li class="li">
+                     <label for="email">
+                         <input id="email"  type="email" placeholder="loremipsum@gmail.com" required>
+                     </label>
+                 </li>
+                 <li class="li">
+                     <textarea name="comments" id="comments" cols="30" rows="10" placeholder="write your message here..." required></textarea>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] Kindly make sure that Message input has `maxlength="500"` attribute. So Kindly Fix it.

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @MPDADDY  @denigit5  , 

Great job for implemented  the some of the Project Requirement , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻
✔️ No Linter errors
✔️ Correct Git Hub-Flow
✔️ Good commit messages
✔️ Professional README

almost

## Required Changes ♻️
_Check the comments below the review._


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @MPDADDY & @denigit5,

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by MPDADDY** [comment]
The mobile version of the portfolio project is comlete.

**Comment by MPDADDY** [comment]
I want to thank all reviewers for great reviews youv'e given to this project.

**Pull request merged**